doc/cabal-project.rst

```diff
@@ -194,8 +194,8 @@ Specifying Packages from Remote Version Control Locations
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Starting with Cabal 2.4, there is now a stanza
-``source-repository-package`` for specifying packages from an external
-version control.
+``source-repository-package`` for specifying remote packages from a version control system that cabal should consider during package retrieval.
```

Comment:
Improvement!

I'm not a fan of "remote package" because it's ambiguous: it may mean non-local, or package stored somewhere, or something else. In particular, changing the old `for specifying packages from ...` to `for specifying remote packages from...` is a regression.

I'd say:
```
for specifying packages stored in a remote version control system that cabal should consider during package retrieval.
```